### DISCUSSION TITLE
Multiple enable requests

### DISCUSSION THREAD

**Discussion by easiwriter** [pointfreeco/swift-composable-architecture#3393] [Q&A]
When I launch a Composable Architecture project in Xcode I receive multiple requests to enable package items. Is there a way of suppressing this?

**Comment by mbrandonw** [comment]
Hi @easiwriter, yes you can skip validation of all macros by running this from the command line:

```
defaults write com.apple.dt.Xcode IDESkipMacroFingerprintValidation -bool YES
```

**Comment by easiwriter** [comment]
Brilliant. Thanks for this.
